.github/workflows/ci.yml

```diff
@@ -29,10 +29,10 @@ jobs:
       - name: Build and Test
         run: Scripts/build.swift xcode ${{ matrix.platform }} `which xcpretty`
       - name: Upload Results
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: Test Results
+          name: Test Results (${{ matrix.platform }})
```

Comment:
`upload-artifact` no longer supports uploading multiple artifacts with the same name